ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -29,6 +29,16 @@ r, ".* ERR kernel:.* Module gpio_ich is blacklisted.*"
 r, ".* skipping since it causes crash: SAI_STP_ATTR_BRIDGE_ID.*"
 r, ".* ERR monit.*Expected containers not running: telemetry.*"
 
+# Errors for config reload on broadcom platform on 202311
+r, ".* ERR swss#orchagent: :- queryHashNativeHashFieldListEnumCapabilities: Failed to get attribute.*"
+r, ".* ERR swss#orchagent: :- queryHashNativeHashFieldListAttrCapabilities: Failed to get attribute.*"
+r, ".* ERR swss#orchagent: :- querySwitchEcmpHashAlgorithmEnumCapabilities: Failed to get attribute.*"
+r, ".* ERR swss#orchagent: :- querySwitchLagHashAlgorithmEnumCapabilities: Failed to get attribute.*"
+r, ".* ERR syncd\d*#syncd.*SAI_API_PORT:_brcm_sai_read_fec_stat_err_counters.* failed with error Feature unavailable.*"
```

Comment:
@byu343 for the 5 lines above I understand they are caused by https://github.com/sonic-net/SONiC/pull/1101 added by Nvidia. What I am not sure is the 3 lines below, Could you please clarify why you want to ignore them? What is the real root cause for those 3 lines?

Reply:
r, ".* ERR syncd\d*#syncd.*SAI_API_PORT:_brcm_sai_read_fec_stat_err_counters.* failed with error Feature unavailable.*"
r, ".* ERR syncd\d*#syncd.*SAI_API_PORT:brcm_sai_get_port_stats.* port fdr stats get failed with error Feature unavailable.*"
Above two lines are from broadcom SAI for reading FEC counters on certain ports not supporting them.

r, ".* ERR swss#portsyncd: :- readData: netlink reports an error=-33 on reading a netlink socket.*"
r, ".* ERR systemd-udevd\[\d*\]:.* bcm0: Failed to get link config: No such device.*"
We only see occasional issues with them from our testing results.  I should remove them from the this change